### PR TITLE
Add detail documentation of inverse kinematics

### PR DESCRIPTION
**Pull request by Naoki-Hiraoka**
https://github.com/euslisp/jskeus/pull/541#issuecomment-508461229

I added detail documentations about : inverse-kinematics to 
```irtmodel.l```, ```irtrobot.l```,

and added documentations about :fullbody-inverse-kinematics and :calc-static-balance-point to
```irtmodel.tex```.

### CODE REVIEW

**Comment by k-okada** [comment]
Thank you for contributing jskeus documentation<br>
Please check latest documents before merging<br>

PDF version of Japanese jmanual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/253/artifacts/0//home/circleci/project/artifacts/jmanual.pdf' target='_blank'>jmanual.pdf</a></strong>
HTML version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/253/artifacts/0//home/circleci/project/artifacts/html/jmanual.html' target='_blank'>jmanual.html</a></strong>
Sphinx (ReST) version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/253/artifacts/0//home/circleci/project/artifacts/rst/html/jmanual.html' target='_blank'>jmanual.rst</a></strong>

**Comment by k-okada** [comment]
Thank you for contributing jskeus documentation<br>
Please check latest documents before merging<br>

PDF version of Japanese jmanual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/257/artifacts/0//home/circleci/project/artifacts/jmanual.pdf' target='_blank'>jmanual.pdf</a></strong>
HTML version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/257/artifacts/0//home/circleci/project/artifacts/html/jmanual.html' target='_blank'>jmanual.html</a></strong>
Sphinx (ReST) version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/257/artifacts/0//home/circleci/project/artifacts/rst/html/jmanual.html' target='_blank'>jmanual.rst</a></strong>

**Comment by k-okada** [comment]
Thank you for contributing jskeus documentation<br>
Please check latest documents before merging<br>

PDF version of Japanese jmanual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/261/artifacts/0//home/circleci/project/artifacts/jmanual.pdf' target='_blank'>jmanual.pdf</a></strong>
HTML version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/261/artifacts/0//home/circleci/project/artifacts/html/jmanual.html' target='_blank'>jmanual.html</a></strong>
Sphinx (ReST) version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/261/artifacts/0//home/circleci/project/artifacts/rst/html/jmanual.html' target='_blank'>jmanual.rst</a></strong>

File: doc/irtmodel.tex [hunk 1]
```diff
@@ -2017,6 +2017,149 @@ \subsubsection{inverse-kinematicsのtarget-coordsに関数を指定する例}
...
+\begin{verbatim}
+:target-centroid-pos (midpoint 0.5 (send *robot* :rleg :end-coords :worldpos)
+                                   (send *robot* :lleg :end-coords :worldpos))
+:cog-translation-axis :z)
+\end{verbatim}
+}
+の行では重心の逆運動学を指定しており、\verb|:cog-translation-axis :z|で
+z方向の重心の移動は許した状態で\verb|:target-centroid-pos|で目標位置として
+両足の中間の座標を与えることによって、重心のxy座標を両足の中間に一致させる
+条件のもとで逆運動学を解くことになる。
```
Comment (mmurooka) [inline doc/irtmodel.tex]: この2つの引数は，デフォルト値をそのまま与えてるかな？
その場合は，「これらの引数は，デフォルト値になっているので，省略可能である．」と書いておくのが良い気がします．

File: doc/irtmodel.tex [hunk 2]
```diff
@@ -2017,6 +2017,149 @@ \subsubsection{inverse-kinematicsのtarget-coordsに関数を指定する例}
...
+               (send *robot* :link-list (send *robot* :larm :end-coords :parent)))
+         :translation-axis (list t t t)
+         :rotation-axis (list t t nil)
+         :target-centroid-pos (midpoint 0.5 (send *robot* :rleg :end-coords :worldpos)
+                                            (send *robot* :lleg :end-coords :worldpos))
+         :cog-translation-axis :z)
+      (send *obj* :locate (send *robot* :centroid) :world)
+      (send *irtviewer* :draw-objects))
+\end{verbatim}
+}
```
Comment (mmurooka) [inline doc/irtmodel.tex]:
```
;; inverse-kinematics failed.
;; dif-pos : #f(0.525164 -0.432807 0.4519)/(0.816904/1)
;; dif-rot : #f(1.856843e-05 3.874633e-05 -4.113339e-05)/(5.948126e-05/0.017453)
;; dif-pos : #f(0.785245 -2.68854 8.69733)/(9.1372/1)
;; dif-rot : #f(0.000157 0.000699 0.000129)/(0.000728/0.017453)
;; dif-pos : #f(12.8136 -0.652139 0.598285)/(12.8442/1)
;; dif-rot : #f(0.0 0.0 0.0)/(0.0/0.017453)
;; cog-dif : #f(-10.0298 4.88267)/(11.1552/5.0)
;;  coords : #<coordinates #X6a6af38  39.29 -46.783 -138.142 / -0.206 0.195 -0.307>
;;  angles : (#f(-2.0726 1.66877 -6.08552 -1.13525 -0.780192 -0.318112) 0.0 0.0 0.0 0.0 10.0 20.0 0.0 -20.0 10.0 0.0 0.0 10.0 20.0 0.0 -20.0 10.0 0.0 0.0 0.0 0.0 -15.0 30.0 -15.0 0.0 0.0 0.0 -15.0 30.0 -15.0 0.0)
;;    args : ((#<coordinates #X72dcb68  0.0 -75.0 -695.463 / 0.0 0.0 0.0> #<coordinates #X71975b8  0.0 75.0 -695.463 / 0.0 0.0 0.0> #<coordinates #X7197678  400.0 100.0 -600.0 / 0.0 0.0 0.0>) :move-target (#<cascaded-coords #X71fe130  -0.525 -74.567 -695.915 / 4.113e-05 -3.875e-05 -1.857e-05> #<cascaded-coords #X71fe370  -0.779 77.687 -704.161 / -0.0 -0.001 -0.0> #<cascaded-coords #X7443c70  391.183 102.817 -591.095 / -0.229 0.805 0.164>) :link-list ((#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X747e248 :rleg-link0  -7.408 -156.383 -256.176 / 0.06 0.269 -0.246> #<bodyset-link #X74b5d30 :rleg-link1  -7.408 -156.383 -256.176 / 0.06 0.269 0.222> #<bodyset-link #X74dd830 :rleg-link2  -7.408 -156.383 -256.176 / -0.179 -0.687 0.278> #<bodyset-link #X68912d8 :rleg-link3  154.87 -116.067 -442.027 / 0.167 0.653 0.27> #<bodyset-link #X6890558 :rleg-link4  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 0.214> #<bodyset-link #X688f4a8 :rleg-link5  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 -1.857e-05>) (#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X688ea58 :lleg-link0  13.283 -14.635 -300.669 / 0.076 0.272 -0.241> #<bodyset-link #X688e728 :lleg-link1  13.283 -14.635 -300.669 / 0.076 0.272 0.275> #<bodyset-link #X688d480 :lleg-link2  13.283 -14.635 -300.669 / -0.237 -0.714 0.353> #<bodyset-link #X688d288 :lleg-link3  182.866 33.384 -477.97 / 0.276 0.788 0.38> #<bodyset-link #X688cf58 :lleg-link4  -0.822 77.697 -641.661 / -0.0 -0.001 0.264> #<bodyset-link #X71da438 :lleg-link5  -0.822 77.697 -641.661 / -0.0 -0.001 -0.0>) (#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X71187e0 :torso-link0  36.261 -50.01 -149.832 / -0.948 -0.073 -0.355> #<bodyset-link #X7123590 :torso-link1  36.261 -50.01 -149.832 / 0.765 -1.212 -1.73> #<bodyset-link #X7300a10 :larm-link0  44.579 181.71 -212.227 / -0.686 -0.657 -0.453> #<bodyset-link #X732ede0 :larm-link1  44.579 181.71 -212.227 / -0.686 -0.657 0.309> #<bodyset-link #X73695b0 :larm-link2  44.579 181.71 -212.227 / -0.655 -0.665 0.29> #<bodyset-link #X73a49c8 :larm-link3  221.444 145.085 -419.596 / -0.691 -0.752 0.314> #<bodyset-link #X73d4ef8 :larm-link4  357.444 110.681 -555.042 / -0.464 -0.791 0.155> #<bodyset-link #X7403208 :larm-link5  357.444 110.681 -555.042 / -0.464 -0.791 0.162> #<bodyset-link #X7446e80 :larm-link6  357.444 110.681 -555.042 / -0.455 -0.753 0.156>)) :move-target (#<cascaded-coords #X71fe130  -0.525 -74.567 -695.915 / 4.113e-05 -3.875e-05 -1.857e-05> #<cascaded-coords #X71fe370  -0.779 77.687 -704.161 / -0.0 -0.001 -0.0> #<cascaded-coords #X7443c70  391.183 102.817 -591.095 / -0.229 0.805 0.164>) :link-list ((#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X747e248 :rleg-link0  -7.408 -156.383 -256.176 / 0.06 0.269 -0.246> #<bodyset-link #X74b5d30 :rleg-link1  -7.408 -156.383 -256.176 / 0.06 0.269 0.222> #<bodyset-link #X74dd830 :rleg-link2  -7.408 -156.383 -256.176 / -0.179 -0.687 0.278> #<bodyset-link #X68912d8 :rleg-link3  154.87 -116.067 -442.027 / 0.167 0.653 0.27> #<bodyset-link #X6890558 :rleg-link4  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 0.214> #<bodyset-link #X688f4a8 :rleg-link5  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 -1.857e-05>) (#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X688ea58 :lleg-link0  13.283 -14.635 -300.669 / 0.076 0.272 -0.241> #<bodyset-link #X688e728 :lleg-link1  13.283 -14.635 -300.669 / 0.076 0.272 0.275> #<bodyset-link #X688d480 :lleg-link2  13.283 -14.635 -300.669 / -0.237 -0.714 0.353> #<bodyset-link #X688d288 :lleg-link3  182.866 33.384 -477.97 / 0.276 0.788 0.38> #<bodyset-link #X688cf58 :lleg-link4  -0.822 77.697 -641.661 / -0.0 -0.001 0.264> #<bodyset-link #X71da438 :lleg-link5  -0.822 77.697 -641.661 / -0.0 -0.001 -0.0>) (#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #<bodyset-link #X71187e0 :torso-link0  36.261 -50.01 -149.832 / -0.948 -0.073 -0.355> #<bodyset-link #X7123590 :torso-link1  36.261 -50.01 -149.832 / 0.765 -1.212 -1.73> #<bodyset-link #X7300a10 :larm-link0  44.579 181.71 -212.227 / -0.686 -0.657 -0.453> #<bodyset-link #X732ede0 :larm-link1  44.579 181.71 -212.227 / -0.686 -0.657 0.309> #<bodyset-link #X73695b0 :larm-link2  44.579 181.71 -212.227 / -0.655 -0.665 0.29> #<bodyset-link #X73a49c8 :larm-link3  221.444 145.085 -419.596 / -0.691 -0.752 0.314> #<bodyset-link #X73d4ef8 :larm-link4  357.444 110.681 -555.042 / -0.464 -0.791 0.155> #<bodyset-link #X7403208 :larm-link5  357.444 110.681 -555.042 / -0.464 -0.791 0.162> #<bodyset-link #X7446e80 :larm-link6  357.444 110.681 -555.042 / -0.455 -0.753 0.156>)) :cog-gain 1.0 :centroid-thre 5.0 :target-centroid-pos #f(0.0 0.0 -695.463) :cog-translation-axis :z :centroid-offset-func nil :cog-null-space nil :min-loop 2 :additional-weight-list ((#<bodyset-link #X70fa2c0 :waist  39.29 -46.783 -138.142 / -0.206 0.195 -0.307> #f(0.1 0.1 0.1 0.1 0.5 0.5))) :move-target (#<cascaded-coords #X71fe130  -0.525 -74.567 -695.915 / 4.113e-05 -3.875e-05 -1.857e-05> #<cascaded-coords #X71fe370  -0.779 77.687 -704.161 / -0.0 -0.001 -0.0> #<cascaded-coords #X7443c70  391.183 102.817 -591.095 / -0.229 0.805 0.164>) :link-list ((#<bodyset-link #X747e248 :rleg-link0  -7.408 -156.383 -256.176 / 0.06 0.269 -0.246> #<bodyset-link #X74b5d30 :rleg-link1  -7.408 -156.383 -256.176 / 0.06 0.269 0.222> #<bodyset-link #X74dd830 :rleg-link2  -7.408 -156.383 -256.176 / -0.179 -0.687 0.278> #<bodyset-link #X68912d8 :rleg-link3  154.87 -116.067 -442.027 / 0.167 0.653 0.27> #<bodyset-link #X6890558 :rleg-link4  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 0.214> #<bodyset-link #X688f4a8 :rleg-link5  -0.528 -74.566 -633.415 / 4.113e-05 -3.875e-05 -1.857e-05>) (#<bodyset-link #X688ea58 :lleg-link0  13.283 -14.635 -300.669 / 0.076 0.272 -0.241> #<bodyset-link #X688e728 :lleg-link1  13.283 -14.635 -300.669 / 0.076 0.272 0.275> #<bodyset-link #X688d480 :lleg-link2  13.283 -14.635 -300.669 / -0.237 -0.714 0.353> #<bodyset-link #X688d288 :lleg-link3  182.866 33.384 -477.97 / 0.276 0.788 0.38> #<bodyset-link #X688cf58 :lleg-link4  -0.822 77.697 -641.661 / -0.0 -0.001 0.264> #<bodyset-link #X71da438 :lleg-link5  -0.822 77.697 -641.661 / -0.0 -0.001 -0.0>) (#<bodyset-link #X71187e0 :torso-link0  36.261 -50.01 -149.832 / -0.948 -0.073 -0.355> #<bodyset-link #X7123590 :torso-link1  36.261 -50.01 -149.832 / 0.765 -1.212 -1.73> #<bodyset-link #X7300a10 :larm-link0  44.579 181.71 -212.227 / -0.686 -0.657 -0.453> #<bodyset-link #X732ede0 :larm-link1  44.579 181.71 -212.227 / -0.686 -0.657 0.309> #<bodyset-link #X73695b0 :larm-link2  44.579 181.71 -212.227 / -0.655 -0.665 0.29> #<bodyset-link #X73a49c8 :larm-link3  221.444 145.085 -419.596 / -0.691 -0.752 0.314> #<bodyset-link #X73d4ef8 :larm-link4  357.444 110.681 -555.042 / -0.464 -0.791 0.155> #<bodyset-link #X7403208 :larm-link5  357.444 110.681 -555.042 / -0.464 -0.791 0.162> #<bodyset-link #X7446e80 :larm-link6  357.444 110.681 -555.042 / -0.455 -0.753 0.156>)) :translation-axis (t t t) :rotation-axis (t t nil) :target-centroid-pos #f(0.0 0.0 -695.463) :debug-view :no-message :cog-translation-axis :z)
;; command : (let ((r (instance sample-robot :init))) (progn (send r :newcoords (make-coords :4x4 #2f((0.960333 0.137937 0.242352 39.2903) (-0.200886 0.944982 0.258176 -46.7827) (-0.193406 -0.29662 0.935206 -138.142) (0.0 0.0 0.0 1.0)))) (mapc #'(lambda (j a) (send* j :joint-angle a nil)) (list (send r :joint119320032) (send r :torso-waist-y) (send r :torso-waist-p) (send r :head-neck-y) (send r :head-neck-p) (send r :larm-shoulder-p) (send r :larm-shoulder-r) (send r :larm-shoulder-y) (send r :larm-elbow-p) (send r :larm-wrist-y) (send r :larm-wrist-r) (send r :larm-wrist-p) (send r :rarm-shoulder-p) (send r :rarm-shoulder-r) (send r :rarm-shoulder-y) (send r :rarm-elbow-p) (send r :rarm-wrist-y) (send r :rarm-wrist-r) (send r :rarm-wrist-p) (send r :lleg-crotch-y) (send r :lleg-crotch-r) (send r :lleg-crotch-p) (send r :lleg-knee-p) (send r :lleg-ankle-p) (send r :lleg-ankle-r) (send r :rleg-crotch-y) (send r :rleg-crotch-r) (send r :rleg-crotch-p) (send r :rleg-knee-p) (send r :rleg-ankle-p) (send r :rleg-ankle-r)) '(#f(-2.0726 1.66877 -6.08552 -1.13525 -0.780192 -0.318112) 0.0 0.0 0.0 0.0 10.0 20.0 0.0 -20.0 10.0 0.0 0.0 10.0 20.0 0.0 -20.0 10.0 0.0 0.0 0.0 0.0 -15.0 30.0 -15.0 0.0 0.0 0.0 -15.0 30.0 -15.0 0.0)) (objects (list r))) (send* r :inverse-kinematics (list (list (make-coords :pos #f(0.0 -75.0 -695.463) :rot #2f((1.0 0.0 0.0) (0.0 1.0 0.0) (0.0 0.0 1.0))) (make-coords :pos #f(0.0 75.0 -695.463) :rot #2f((1.0 0.0 0.0) (0.0 1.0 0.0) (0.0 0.0 1.0))) (make-coords :pos #f(400.0 100.0 -600.0) :rot #2f((1.0 0.0 0.0) (0.0 1.0 0.0) (0.0 0.0 1.0)))) :dump-command nil :debug-view t :move-target (list (let* ((p (send r :link :rleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 -5.472834e-22 1.110223e-16) (0.0 1.0 0.0 0.0) (5.472834e-22 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :lleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 7.719249e-21 0.0) (0.0 1.0 0.0 -1.421085e-14) (-7.719249e-21 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :larm-link6)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((2.220446e-16 0.0 1.0 0.0) (0.0 1.0 0.0 2.842171e-14) (-1.0 0.0 2.220446e-16 -50.0) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt)) :link-list (list (list (send r :link :waist) (send r :link :rleg-link0) (send r :link :rleg-link1) (send r :link :rleg-link2) (send r :link :rleg-link3) (send r :link :rleg-link4) (send r :link :rleg-link5)) (list (send r :link :waist) (send r :link :lleg-link0) (send r :link :lleg-link1) (send r :link :lleg-link2) (send r :link :lleg-link3) (send r :link :lleg-link4) (send r :link :lleg-link5)) (list (send r :link :waist) (send r :link :torso-link0) (send r :link :torso-link1) (send r :link :larm-link0) (send r :link :larm-link1) (send r :link :larm-link2) (send r :link :larm-link3) (send r :link :larm-link4) (send r :link :larm-link5) (send r :link :larm-link6))) :move-target (list (let* ((p (send r :link :rleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 -5.472834e-22 1.110223e-16) (0.0 1.0 0.0 0.0) (5.472834e-22 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :lleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 7.719249e-21 0.0) (0.0 1.0 0.0 -1.421085e-14) (-7.719249e-21 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :larm-link6)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((2.220446e-16 0.0 1.0 0.0) (0.0 1.0 0.0 2.842171e-14) (-1.0 0.0 2.220446e-16 -50.0) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt)) :link-list (list (list (send r :link :waist) (send r :link :rleg-link0) (send r :link :rleg-link1) (send r :link :rleg-link2) (send r :link :rleg-link3) (send r :link :rleg-link4) (send r :link :rleg-link5)) (list (send r :link :waist) (send r :link :lleg-link0) (send r :link :lleg-link1) (send r :link :lleg-link2) (send r :link :lleg-link3) (send r :link :lleg-link4) (send r :link :lleg-link5)) (list (send r :link :waist) (send r :link :torso-link0) (send r :link :torso-link1) (send r :link :larm-link0) (send r :link :larm-link1) (send r :link :larm-link2) (send r :link :larm-link3) (send r :link :larm-link4) (send r :link :larm-link5) (send r :link :larm-link6))) :cog-gain 1.0 :centroid-thre 5.0 :target-centroid-pos #f(0.0 0.0 -695.463) :cog-translation-axis :z :centroid-offset-func nil :cog-null-space nil :min-loop 2 :additional-weight-list (list (list (send r :link :waist) #f(0.1 0.1 0.1 0.1 0.5 0.5))) :move-target (list (let* ((p (send r :link :rleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 -5.472834e-22 1.110223e-16) (0.0 1.0 0.0 0.0) (5.472834e-22 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :lleg-link5)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((1.0 0.0 7.719249e-21 0.0) (0.0 1.0 0.0 -1.421085e-14) (-7.719249e-21 0.0 1.0 -62.5) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt) (let* ((p (send r :link :larm-link6)) (mt (make-cascoords :coords (send (send p :copy-worldcoords) :transform (make-coords :4x4 #2f((2.220446e-16 0.0 1.0 0.0) (0.0 1.0 0.0 2.842171e-14) (-1.0 0.0 2.220446e-16 -50.0) (0.0 0.0 0.0 1.0))))))) (send p :assoc mt) mt)) :link-list (list (list (send r :link :rleg-link0) (send r :link :rleg-link1) (send r :link :rleg-link2) (send r :link :rleg-link3) (send r :link :rleg-link4) (send r :link :rleg-link5)) (list (send r :link :lleg-link0) (send r :link :lleg-link1) (send r :link :lleg-link2) (send r :link :lleg-link3) (send r :link :lleg-link4) (send r :link :lleg-link5)) (list (send r :link :torso-link0) (send r :link :torso-link1) (send r :link :larm-link0) (send r :link :larm-link1) (send r :link :larm-link2) (send r :link :larm-link3) (send r :link :larm-link4) (send r :link :larm-link5) (send r :link :larm-link6))) :translation-axis (list t t t) :rotation-axis (list t t nil) :target-centroid-pos #f(0.0 0.0 -695.463) :debug-view :no-message :cog-translation-axis :z)))
;; dump debug command to /tmp/irtmodel-ik-23317/sample-robot-2019-07-08-15-03-32-18-05-81-failure.l
;; (progn (load "/tmp/irtmodel-ik-23317/sample-robot-2019-07-08-15-03-32-18-05-81-failure.l")(ik-setup)(ik-check))
```

**Comment by Naoki-Hiraoka** [comment]
Thank you for comments.

I removed pdf,dvi from commits, and fix the document and example as suggested.

File: doc/irtmodel.tex (hunk 1 above)
Comment (Naoki-Hiraoka) [reply doc/irtmodel.tex]: デフォルト値である旨追加しました

File: doc/irtmodel.tex (hunk 2 above)
Comment (Naoki-Hiraoka) [reply doc/irtmodel.tex]: 例を修正しました

**Comment by k-okada** [comment]
Thank you for contributing jskeus documentation<br>
Please check latest documents before merging<br>

PDF version of Japanese jmanual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/273/artifacts/0//home/circleci/project/artifacts/jmanual.pdf' target='_blank'>jmanual.pdf</a></strong>
HTML version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/273/artifacts/0//home/circleci/project/artifacts/html/jmanual.html' target='_blank'>jmanual.html</a></strong>
Sphinx (ReST) version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/273/artifacts/0//home/circleci/project/artifacts/rst/html/jmanual.html' target='_blank'>jmanual.rst</a></strong>

**Comment by mmurooka** [comment]
このPRは他のも含んでいる？
https://github.com/euslisp/jskeus/pull/549/commits/907b36e93ff449be1586f5258dfbfd66eeca6707 のコミットとかはどこか別のところでも見た気もするけれど．

**Comment by Naoki-Hiraoka** [comment]
失礼しました。
余計なcommitを除きました

**Comment by k-okada** [comment]
Thank you for contributing jskeus documentation<br>
Please check latest documents before merging<br>

PDF version of Japanese jmanual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/277/artifacts/0//home/circleci/project/artifacts/jmanual.pdf' target='_blank'>jmanual.pdf</a></strong>
HTML version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/277/artifacts/0//home/circleci/project/artifacts/html/jmanual.html' target='_blank'>jmanual.html</a></strong>
Sphinx (ReST) version of Japanese manual: <strong><a href='https://circleci.com/gh/euslisp/jskeus/277/artifacts/0//home/circleci/project/artifacts/rst/html/jmanual.html' target='_blank'>jmanual.rst</a></strong>

**Pull request merged**